.gitignore

```diff
@@ -1,6 +1,7 @@
-dist
```

Comment:
compilation folder "dist" should be in git ignore

Reply:
dist folder removed

Reply:
Still this PR is adding contents of dist folder to git. I believe this dist folder should be in gitignore to avoid having these files in source control

Reply:
Ah, it was github cache